task-2/index.js

```diff
@@ -1,4 +1,5 @@
-const fs = require('fs')
+import * as fs from 'node:fs';
+
 const csv = require('csvtojson');
```

Comment:
Missed converting to ES6 for this import

Reply:
Done